cmd/config/internal/commands/run-fns.go

```diff
@@ -29,11 +34,12 @@ func GetRunFnRunner(name string) *RunFnRunner {
 		&r.DryRun, "dry-run", false, "print results to stdout")
 	r.Command.Flags().BoolVar(
 		&r.GlobalScope, "global-scope", false, "set global scope for functions.")
-	r.Command.Flags().StringSliceVar(
-		&r.FnPaths, "fn-path", []string{},
-		"directories containing functions without configuration")
-	r.Command.AddCommand(XArgsCommand())
-	r.Command.AddCommand(WrapCommand())
+	r.Command.Flags().StringSliceVarP(
+		&r.FnPaths, "fn-path", "p", []string{},
+		"read functions from these directories instead of the configuration directory.")
+	r.Command.Flags().StringVarP(
+		&r.Image, "image", "i", "",
```

Comment:
-i short flag is overloaded (-i in docker CLI, and denotes 'input ' n other CLIs).

Can we change it to something else? Maybe '-m'?

Reply:
I'll just drop the shorthand, it isn't necessary right now.